style.css

```diff
@@ -529,3 +529,212 @@ p {
 a {
   text-decoration: none;
 }
+
+.footer {
+  background: #e5e5e5;
+}
+
+.containers {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 90px;
+  position: absolute;
+  width: 360px;
+  height: 844px;
+  left: 0;
+  top: 6038px;
+  background: #e5e5e5;
+}
+
+.textheads {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] Kindly make sure you stick to the design as possible as you can. the `typo`, the `padding`, and `margin` must match with the template provided in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193680129-322957cd-c3ac-4730-839e-f204d05fdae1.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193680248-00dfbea1-da09-4598-a190-452b7762c55b.png)